### DISCUSSION TITLE
Import favorites improvements

### DISCUSSION THREAD

**Discussion by airald** [devemux86/cruiser#408] [Q&A] [answered]
When Import favourites, first I delete all groups. A default favourite name appears after delete last group (I suppose should be one gropu at least). When import favourites, if there's an empty group, it's not imported on new device. Instead of it, appears the default fav group name with default name and icon.

As I use first group as "auto POI" container, as simple as have a "dummy" POI that never will be classified in another group and I delete "default" grup name after importing. I know that other users do the same with first group, I wrote it here as knowledge base.

**Comment by devemux86** [comment]
@airald thanks for the description.

At least one group must exist for new favorites to be saved successfully.

Is there a workflow described that we can improve?

**Comment by devemux86** [comment]
Quick favorites in navigation / follow: we could have a fixed group?
That cannot be deleted? Probably with fixed name to recognize it?

Although this could be complicated for users to understand.

**Reply by airald** [reply]
Maybe a default group used to store POIs with new compass function should be a good solution...

Or simply import empty groups.

**Comment by devemux86** [comment]
> Or simply import empty groups.

It will be available in the next update. :slightly_smiling_face:

**Comment by devemux86** [comment]
> Or simply import empty groups.

@airald you can test in:

- https://github.com/devemux86/cruiser/discussions/414

**Reply by airald** [reply]
Empty groups are imported. But if there's a group with the default name, there are no favourites inside it. 

I create the group with 2 favourites, then backup. I delete one favourite. When Import last backup, there's only one favourite.

**Reply by airald** [reply]
I'm trying to reproduce it again and now it seems that a default favourites name is imported correcly, just there are the default created group and one new that contains the saved favourites. I tried yesterday night several times, but today the behaviour is the expected.

**Reply by devemux86** [reply]
@airald thanks for the tests!

Is there anything else to check here?

**Reply by airald** [reply]
All seems OK!

**Comment by devemux86** [comment, marked as answer]
All discussed improvements are implemented in [Cruiser 3.0.38](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
